package.json

```diff
@@ -58,11 +58,12 @@
     "@typescript-eslint/eslint-plugin": "^6.19.0",
     "@typescript-eslint/parser": "^6.19.0",
     "eslint": "^8.56.0",
-    "eslint-config-expensify": "^2.0.43",
+    "eslint-config-expensify": "^2.0.44",
     "eslint-config-prettier": "^8.8.0",
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-jsx-a11y": "^6.8.0",
     "eslint-plugin-react": "^7.31.10",
+    "eslint-plugin-react-hooks": "^4.6.0",
```

Comment:
You can remove this plugin now, please double check `eslint-plugin-react-hooks` from eeslint-config-expensify works 🫣

Reply:
Seems to be working 🙂